.github/workflows/eval.yml

```diff
@@ -68,6 +68,9 @@ jobs:
     strategy:
       matrix:
         system: ${{ fromJSON(needs.attrs.outputs.systems) }}
+        subpackages:
+          - null
+          - pkgsLLVM
```

Comment:
I really don't think we should use this file to specify what should be evaluated! What motivates this instead of `recurseIntoAttrs`?

Reply:
My idea was that it would only do the top-level in 1 job and pkgsLLVM in another.

Reply:
I've dropped this change